### ISSUE TITLE
What is the current status of this proposal?

### ISSUE THREAD

**Issue by oovm** [WebAssembly/wasi-url#2]
Is this proposal still active?

Can we provide some url related string parsing and field definition wit files?

Then I can provide guest implementation in my library

**Comment by sunfishcode** [comment]
No, at this time, wasi-url is not under active development. If you'd like to work on it, perhaps the best thing to do would be to post an outline of what you'd like to see here, and/or create a WASI meeting agenda item to discuss this, and then we can figure out a plan from there.
